rfcs/20210519-single-pass-bnorm/README.md

```diff
@@ -55,42 +92,41 @@ large in relation to result of the formula. In order to minimize rounding
 errors:
 - Sum and SumOfSquares will be stored in fp32 precision, regardless of input
 	data type
-- Kahan summation algorithm will be used
+- Kahan summation algorithm [[#1]][1] will be used
 
 The last line, max( ) is there to make sure rounding errors won't cause
-variance to go negative. Further in batchnorm calculations square root is taken
+variance to go negative. Further in BN calculations square root is taken
 from variance. Negative value would result in NaN.
 
 New formula is considered experimental for now. By default the old one will be
 used. There needs to be a way for user to choose formula.
```

Comment:
shall this paragraph go away (you clarified this in the above integration section)

Reply:
removed, thanks